test/test_models.py

```diff
@@ -213,7 +227,15 @@ def compute_mean_std(tensor):
                 out = model(model_input)
                 # See autocast_flaky_numerics comment at top of file.
                 if name not in autocast_flaky_numerics:
-                    check_out(out)
+                    full_validation &= check_out(out)
+
+        if not full_validation:
```

Comment:
If any of the validations was partial, flag the test as skipped and raise a warning. It's better to flag it as skipped than mark it as green.